### ISSUE TITLE
IPv6 support

### ISSUE THREAD

**Issue by lpereira** [lpereira/hardinfo#374]
- Network interfaces should list their IPv6 addresses
- Change "ARP table" to "Neighbors", and list both the ARP table and the IPv6 NDP (`ip neigh`)

**Comment by baryluk** [comment]
I think this is a serious issue.

`Interfaces` show nothing now (it can be read via `ip address`, or `ip -6 address`). 

`Connections` shows IPv6 connections, but the port is just concatenated, making it tricky to read. It should use `ss -an`, which shows it IPv6 with ports in brackets, similar to URI in HTTP, like this:

`tcp   ESTAB  0      0      [2a02:1:1:1:2:d3:14:67]:59123      [2606:4700:20::1:2]:https        `

(or better with `ss -n`, `[2606:4700:20::1:2]:443`). `ss -apn` is also option to get extra info about sockets.

Routing, shows no routing tables for IPv6 or multicast. These can be accessed via `ip -6 route` and `ip -6 mroute`.

**Comment by lpereira** [comment]
Yes, IPv6 is one of the things that need to be addressed (no pun intended) before the next release is cut.
